Guided Investigation - Azure WAF SQLI.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.    if isinstance(rule_details_df, pd.DataFrame) and not rule_details_df.empty:
</pre><p>I think I created a utils function to do this (for future ref) msticpy.common.data_utils.df_has_data</p><p>Although the import may be longer to type than the saving if you only have one check!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>